content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md

```diff
@@ -15,7 +15,7 @@ is a property of {{< glossary_tooltip text="Pods" term_id="pod" >}} that *attrac
 a set of {{< glossary_tooltip text="nodes" term_id="node" >}} (either as a preference or a
 hard requirement). _Taints_ are the opposite -- they allow a node to repel a set of pods.
 
-_Tolerations_ are applied to pods, and allow (but do not require) the pods to schedule
+_Tolerations_ are applied to pods, it does not guarantee that pod will get scheduled
```

Comment:
The new text does seem more clear to me than what was already there.  What did you feel was wrong with the existing text?

Reply:
@bradtopol @Huang-Wei please weigh in if this sounds better if we're looking to rephrase the content?

```suggestion
_Tolerations_ are applied to pods. They allow but do not necessarily schedule the pods
```

Reply:
> @dipankardas011, IMHO the meaning of the sentence has been altered with the change you're proposing. While I agree that brackets add complexity, I don't think that rephrasing it in the way you've suggested is helping with simplifying/clarifying the subject better.

Okay will improve

Reply:
@divya-mohan0209 LGTM, but TBH put "do not necessarily" reads better:

```
Tolerations are applied to pods. They allow (but not necessarily) the pods to be scheduled...
```